src/Servers/Kestrel/Transport.MsQuic/src/Internal/MsQuicConstants.cs

```diff
@@ -8,7 +8,11 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Transport.MsQuic.Internal
 {
     internal static class MsQuicConstants
     {
-        private const uint Success = 0;
+        public const uint SuccessConst = 0;
+
+        internal static uint InternalError = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? Windows.InternalError : Linux.InternalError;
+        internal static uint Success = 0;
```

Comment:
Why have Success and SuccesConst when they're both the same type and value? Why make SuccesConst public if it's only referenced from a nested class?